### ISSUE TITLE
Yume does not check the options

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#510]
**Reported by valleegr on 13 Jan 2009 20:17 UTC**

When we try to use an unsupported yume option, it does not output the help, only a weird and useless error message.

```
$ yume --toto
Unexpected error detected (127) while distro-query. Exiting.
```

**Comment by dikim33** [comment]
**Modified by valleegr on 14 Jan 2009 03:47 UTC**

**Comment by dikim33** [comment]
**Comment by valleegr on 14 Jan 2009 04:20 UTC**
(In [7838]) This check-in fixes #511 - We now check the options and display the help 
message if the option is invalid.
